lib/media/track/localvideotrack.js

```diff
@@ -210,9 +210,7 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
    * the <code>mediaStreamTrack</code> property. If you want to listen to events on
    * the MediaStreamTrack directly, please do so in the "started" event handler. Also,
    * the {@link LocalVideoTrack}'s ID is no longer guaranteed to be the same as the
-   * underlying MediaStreamTrack's ID. Additionally, the {@link LocalVideoTrack}'s `processor`
-   * and `processedTrack` properties will be null while the restart is in progress and will
-   * become available again once restart is complete.
```

Comment:
Why is this removed?

Reply:
([VIDEO-4028](https://issues.corp.twilio.com/browse/VIDEO-4028)) The initial implementation of track.restart with VideoProcessor is that, VideoTrack.processor and VideoTrack.processedTrack are null after calling track.restart. This is because track.restart is asynchronous and we were waiting for it to finish before re-adding the processors back.

With this change ([VIDEO-4028](https://issues.corp.twilio.com/browse/VIDEO-4028), see also commit), processor and processedTrack properties now not null during track.restart.